### DISCUSSION TITLE
I am a verified student but I still don't have access to copilot

### DISCUSSION THREAD

**Discussion by PranjalShukla2004** [community/community#60846] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I got verified as a student 3 days ago but I still don't have access to copilot, it even says that I don't need to be reverified . What am I supposed to do?

**Comment by Akash1134** [comment, marked as answer]
Hi @PranjalShukla2004 

Once you are verified, it takes a couple of days to process the request. Since you were verified 3 days ago, please ensure your logged into your verified GitHub student account and then Connect Copilot to Github: Go to the Copilot website and log in. Then, go to the Settings section and connect your [Github account to Copilot](https://copilot.github.com/?WT.mc_id=academic-88217-leestott).

Once finishes, follow these [step by step instructions](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279) to setup Copilot.

**Reply by PranjalShukla2004** [reply]
how am I supposed to connect my copilot to github? I tried logging in and opened settings, went to copilot in the coding, planning and automation section but Idk what to do after that. Also when I go to copilot in the settings there are only 2 options- to buy it for an organization or purchase it individually but as I said, I should have access as a student.
![Screenshot 2023-07-14 213905](https://github.com/community/community/assets/116290756/f87302c5-f60d-443f-a406-111995339628)

**Reply by pog5** [reply]
You need to wait a few days (3-7) until GitHub approves your Copilot access.
*If you found this answer helpful, please mark it as the answer.**
